Dockerfile

```diff
@@ -2,8 +2,8 @@
 # Build with multistage build on jdk-11
 FROM gradle:6.3.0-jdk11 as java_build
 
-ARG MVN_S3_INTERNAL_ACCESS_KEY
-ARG MVN_S3_INTERNAL_SECRET_KEY
+ARG NEXUS_USERNAME
+ARG NEXUS_PASSWORD
```

Comment:
I'm wondering if we should keep this here, as this repo is not used in dependencies at all... or add the repository to stay consistent with having it here, but at the same time, it won't build unless we provide these credentials... WDYT?

Reply:
I'm voting on removing :D